config/src/config/quorum_store_config.rs

```diff
@@ -32,12 +33,13 @@ impl Default for QuorumStoreBackPressureConfig {
             // validator has recieved Proof of Store but has not yet committed.
             // The batches generated by different validators could have common transactions.
             // Roughly, target TPS * commit latency seconds * duplication factor of a transaction in batches.
-            backlog_txn_limit_count: 36_000,
+            backlog_txn_limit_count: 72_000,
```

Comment:
This [forge](https://github.com/aptos-labs/aptos-core/actions/runs/9487115246) run at 5k TPS faces a lot of QS backpressure at 5k TPS workload. So, increased the limits.

Reply:
We will need to revert this change as this makes Forge runs unstable. See https://github.com/aptos-labs/aptos-core/pull/13666 and https://aptos-org.slack.com/archives/C077LR8EKGA/p1718213547242389?thread_ts=1718124334.521749&cid=C077LR8EKGA

Reply:
I reverted back the change. With 12k `backlog_txn_limit_count`, we won't be able to do more than like 2-3k TPS though.